ipn/ipnlocal/profiles.go

```diff
@@ -98,21 +98,21 @@ func (pm *profileManager) matchingProfiles(f func(*ipn.LoginProfile) bool) (out
 	return out
 }
 
-func (pm *profileManager) findProfilesByNodeID(nodeID tailcfg.StableNodeID) []*ipn.LoginProfile {
+func (pm *profileManager) findProfilesByNodeID(controlURL string, nodeID tailcfg.StableNodeID) []*ipn.LoginProfile {
```

Comment:
method name doesn't quite match. either rename method and/or document what controlURL does?

Reply:
Done